example/android/app/src/main/java/iterable/reactnativesdk/example/JwtTokenModule.kt

```diff
@@ -0,0 +1,37 @@
+package iterable.reactnativesdk.example
+
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.Promise
+import com.iterable.IterableJwtGenerator
+import java.time.Duration
+
+class JwtTokenModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
+
+    override fun getName(): String {
+        return NAME
+    }
+
+    @ReactMethod
+    fun generateJwtToken(
```

Comment:
Nit pick: generation logic could be unnecessary as its done on server side. It helps to test the behavior locally quickly. But Devs might get confused if app is where generation should happen.